torchvision/transforms/functional.py

```diff
@@ -573,10 +573,9 @@ def perspective(
             :class:`torchvision.transforms.InterpolationMode`. Default is ``InterpolationMode.BILINEAR``.
             If input is Tensor, only ``InterpolationMode.NEAREST``, ``InterpolationMode.BILINEAR`` are supported.
             For backward compatibility integer values (e.g. ``PIL.Image.NEAREST``) are still acceptable.
-        fill (n-tuple or int or float): Pixel fill value for area outside the rotated
+        fill (sequence or int or float, optional): Pixel fill value for area outside the rotated
```

Comment:
Let's make the docstring more explicit about how it works for PIL, tensor and torchscript:
```python
fill (sequence or int or float, optional): Pixel fill value for the area outside the rotated
    image. If int or float, the value is used for all bands respectively.
    This option is supported for PIL image and Tensor inputs. 
    In torchscript mode single int/float value is not supported, please use a tuple 
    or list of length 1: ``[value, ]``.
    If input is PIL Image, the options is only available for ``Pillow>=5.0.0``.
```

same for other docstrings.

Reply:
@vfdev-5 I did some sorting. It seems the current version means affine functions support:
int/float/list/None for Tensor.
int/float/list/tuple/None for PIL.
list/None for torchscript.
Not sure what I should write in the docs.

Reply:
I thought PIL and Tensor could support the same types. Where is actually the problem with tuple if input is Tensor ?

Reply:
Not very sure, let me test for it a bit. I wrote the code with only list in mind.

Reply:
You are right, torch.tensor() can also convert tuples. There is no functional mismatch between tensor and PIL.